lib/application_configuration.rb

```diff
@@ -13,7 +13,7 @@ def configured_value(config_path, options = {})
     # a query will yield false (as a configured value), so we cannot
     # use standard .present? or .blank?. We also cannot use .empty?
     # because TrueClass and FalseClass do not support it
-    if value != {} then value
+    if !value.nil? then value
```

Comment:
Dig, in fact, returns 'nil' when a missing key is hit:
http://ruby-doc.org/core-2.3.0/Hash.html#method-i-dig